v3/src/components/graph/components/parent-toggles.tsx

```diff
@@ -178,7 +180,8 @@ export const ParentToggles = observer(function ParentToggles() {
       },
       {
         undoStringKey: "DG.mainPage.mainPane.undoButton.toolTip",
-        redoStringKey: "DG.mainPage.mainPane.redoButton.toolTip"
+        redoStringKey: "DG.mainPage.mainPane.redoButton.toolTip",
+        log: isOnlyLastShown ? "Disable last parent toggle" : "Enable last parent toggle"
```

Comment:
Should probably be something more like:
```suggestion
        log: isOnlyLastShown ? "Disable only showing last parent toggle" : "Enable only showing last parent toggle"
```